yucca/run/run_lightning_training.py

```diff
@@ -77,10 +81,10 @@ def main():
     args = parser.parse_args()
 
     task = maybe_get_task_from_task_id(args.task)
-    model = args.m
+    model_name = args.m
```

Comment:
Argparse supports short and long versions such that the ages object has attribute `args.model_name`, but you can still just pass it with flag `yucca_trainer -m something`. Then we can just use the arga object directly instead of "translating" it into a variable. See [this](https://docs.python.org/3/library/argparse.html#name-or-flags) for details.